app/importers/mahonia_mapper.rb

```diff
@@ -28,6 +28,7 @@ def metadata=(meta)
         @metadata[BEPRESS_TERMS_MAP[k.to_sym]] = be_press_hash.delete k
       end
     end
+    MahoniaValidator.new.validate(@metadata)
```

Comment:
If you implement the validator so `#validate` accepts a `parser` argument, you can simply add the validator instance to the parser's `@validators`.

My sense is that we should avoid putting validation in mappers.

Reply:
See parser validation documentation at: https://github.com/curationexperts/darlingtonia/blob/master/lib/darlingtonia/parser.rb#L12-L36